go.mod

```diff
@@ -32,7 +32,7 @@ require (
 	github.com/solo-io/go-utils v0.24.8
 	github.com/solo-io/k8s-utils v0.7.2
 	github.com/solo-io/protoc-gen-ext v0.0.18
-	github.com/solo-io/protoc-gen-openapi v0.1.1
+	github.com/solo-io/protoc-gen-openapi v0.2.4
```

Comment:
nit: When bumping libraries across minor versions, I tend to callout in the PR body what those changes were. Mostly just to get in the habit of validating changes, because we have noticed that sometimes there are behavioral change that impact us. Since this is an internal library that we manage, and is low risk, it's certainly not a requirement (more sharing as it's a pattern I follow)